src/Tizen.NUI/src/public/BaseComponents/ViewAccessibility.cs

```diff
@@ -422,6 +422,24 @@ public void EmitTextCursorMovedEvent(int cursorPosition)
             if (NDalicPINVOKE.SWIGPendingException.Pending) throw NDalicPINVOKE.SWIGPendingException.Retrieve();
         }
 
+        /// <summary>
+        /// Gets or sets a value indicating suppressed events.
+        /// </summary>
```

Comment:
Could you add `[EditorBrowsable(EditorBrowsableState.Never)]` here to silence the ACR bot?

Reply:
Thank you for this meticulous review and below.